TemplateApplication/Schedule/EventContextView.swift

```diff
@@ -29,8 +29,8 @@ struct EventContextView: View {
                             .font(.headline)
                             .accessibilityLabel(
                                 eventContext.event.complete
-                                    ? "COMPLETED_TASK_LABEL \(eventContext.task.title)"
-                                    : "TASK_LABEL \(eventContext.task.title)"
+                                ? "COMPLETED_TASK_LABEL \(eventContext.task.title)"
+                                : "TASK_LABEL \(eventContext.task.title)"
```

Comment:
I think that the intentional indention here makes it a bit easier to read the code.